tests/integration_tests/README.md

```diff
@@ -0,0 +1,19 @@
+These tests will post to a Github PR and read from Velox Conbench. To run them, you
+need these environment variables set. Otherwise, they will be skipped.
+
+- `GITHUB_API_TOKEN` - an API token that can post to https://github.com/austin3dickey/austin-games/pull/1
+- `CONBENCH_EMAIL` and `CONBENCH_PASSWORD` - login information to https://velox-conbench.voltrondata.run
```

Comment:
We shouldn't actually need the authentication here regardless, but could we have this reference the arrow conbench instance? That has a lot more data on it already + ties in with the first originator of this work being for that.

Also, would it be possible to have this reference a conbench repo with the github token?

Reply:
Yeah, good ideas.

Reply:
done and done